pkg/admissioncontroller/webhooks/auth/seed/authorizer.go

```diff
@@ -64,6 +67,8 @@ func (a *authorizer) Authorize(_ context.Context, attrs auth.Attributes) (auth.D
 		switch requestResource {
 		case cloudProfileResource:
```

Comment:
BTW: what about other API versions of the `core.gardener.cloud` API?

Reply:
https://github.com/gardener/gardener/blob/8a51383c03fc647c8231c737c734a9751c3c23cd/pkg/admissioncontroller/webhooks/auth/seed/authorizer.go#L48

Reply:
Ah right, thanks.